main/src/main/java/org/mobilitydata/gtfsvalidator/notice/BlockTripsWithOverlappingStopTimesNotice.java

```diff
@@ -24,7 +28,8 @@ public BlockTripsWithOverlappingStopTimesNotice(
             .put("serviceIdB", serviceIdB)
             .put("blockId", blockId)
             .put("intersection", intersection.toYYYYMMDD())
-            .build());
+            .build(),
+        SeverityLevel.WARNING);
```

Comment:
It is documented as ERROR at the top. Google enforces that as an ERROR, so let's make it an ERROR.

Reply:
Thanks for flagging that.